README.md

```diff
@@ -117,8 +117,39 @@ are working on it.
     git checkout jaseem1
     ```
 3. Run Migration
-    
+
+### By submitting Patches   
```

Comment:
Why not just move this to `CONTRIBUTING.md` ? Populating `README` leads to less readability on homepage.

Reply:
@salmanulfarzy The whole of README seems to only deal with contributing: development and testing. Since we don't have any 'regular users', it doesn't feel too bad to have all content related to contributing in the README.